src/Storages/Kafka/StorageKafka.h

```diff
@@ -142,6 +156,8 @@ class StorageKafka final : public IStorage, WithContext
 
     bool streamToViews();
     bool checkDependencies(const StorageID & table_id);
+
+    std::shared_ptr<const String> rdkafka_stat;
```

Comment:
shouldn't that be per consumer? (instead of global per table).

Reply:
The statistic is global (it is complete for every consumer), I do not see why it is better to have individual copy of the same data in all consumers.
@filimonov  Do you have concerns behind this suggestion?